CommandLine.Tests/HelpViewTests.cs

```diff
@@ -303,5 +303,17 @@ public void When_unmatched_tokens_are_allowed_then_help_view_indicates_it()
 
             helpView.Should().Contain("Usage: some-command [options] [[--] <additional arguments>...]]");
         }
+
+        [Fact]
+        public void When_option_alias_separator_is_provided_it_is_used()
+        {
+            var command = Command("command", "Help text",
+                Option("-l|--list", "Help for list option"),
+                Option("-a|--all", "Help for all option"));
+            HelpViewOptions helpViewOptions = new HelpViewOptions("| ");
+            var helpView = command.HelpView(helpViewOptions);
+            helpView.Should().Contain("-l| --list");
+            helpView.Should().Contain("-a| --all");
```

Comment:
Ok, sounds good. I'll be interested in seeing / hearing about the alternate design.